pinot-spi/src/main/java/org/apache/pinot/spi/data/readers/RecordReaderFactory.java

```diff
@@ -33,18 +33,18 @@
   private static final Map<String, String> DEFAULT_RECORD_READER_CONFIG_CLASS_MAP = new HashMap<>();
 
   // TODO: This could be removed once we have dynamic loading plugins supports.
```

Comment:
@fx19880617 can we remove this now?

Reply:
This requires PluginManager to populate this map, by setting fileFormat and classname when loading each input format plugin.